receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -37,6 +57,52 @@ attributes:
     enum: [dead, live]
 
 metrics:
+  postgresql.bgwriter.buffers.allocated:
+    attributes: []
+    description: Number of buffers allocated.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
```

Comment:
Just double checking - this isn't the real time number of buffers allocated, right?

Reply:
The documentation is a tad spotty, I believe it is a normal counter based off local testing. I also found this discussion that seems to indicate it being a normal counter https://www.postgresql.org/message-id/1a00e942-e7c6-8e4d-afec-4b1befd5688d%40pro-open.de